bower.json

```diff
@@ -12,7 +12,7 @@
   "description": "jQuery Split Pane plugin",
   "license": "MIT",
   "dependencies": {
-    "jquery": "1.11.1"
+    "jquery": ">=1.11.1"
```

Comment:
I think that I should accept minor updates only. So "~1.11.1" is my suggestion.

Are you using jquery version 2?